spec/models/assessment_log_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
```

Comment:
Same as here:
https://github.com/inloco/incognia-demo-api-ruby/pull/30/commits/ec60bc26635686183f2241330567bb8b83238911#r999538241

Reply:
I fixed it but already answered in the other comment